lang/ui.fr.json

```diff
@@ -876,7 +876,7 @@
   },
   "simulator-loud": {
     "defaultMessage": "Loud",
-    "description": "Sound level high threshold marker hover text"
+    "description": "Sound level high threshold marker hover text. This should match the translation of microbit.SoundEvent.LOUD in the api.en.json file but with differing case."
```

Comment:
Should be changed in the ui.en.json file, not ui.fr.json.

Reply:
2d04f073f6ac3c7b0f5ce2edcc13ba4a013b6e13